settings.gradle.kts

```diff
@@ -16,6 +17,10 @@ dependencyResolutionManagement {
     }
 }
 
+plugins {
+    id("com.android.settings") version("8.2.0-beta01")
```

Comment:
can this version be a library version in the version catalog file?

Reply:
Not since last I checked, this settings file is evaluated before version catalogs